tests/tests_geomstats/test_hyperbolic.py

```diff
@@ -18,478 +19,402 @@
-
-        log = self.metric.log(point=point, base_point=base_point)
-
-        result = self.metric.exp(tangent_vec=log, base_point=base_point)
-        expected = point
-        self.assertAllClose(result, expected)
-
-    def test_log_and_exp_general_case_general_dim(self):
-        """
-        Test that the Riemannian exponential
-        and the Riemannian logarithm are inverse.
-
-        Expect their composition to give the identity function.
-        """
-        # Riemannian Log then Riemannian Exp
-        dim = 5
-        n_samples = self.n_samples
-
-        h5 = Hyperboloid(dim=dim)
-        h5_metric = h5.metric
-
-        base_point = h5.random_point()
-        point = h5.random_point()
-        point = gs.cast(point, gs.float64)
-        base_point = gs.cast(base_point, gs.float64)
-        one_log = h5_metric.log(point=point, base_point=base_point)
-
-        result = h5_metric.exp(tangent_vec=one_log, base_point=base_point)
-        expected = point
-        self.assertAllClose(result, expected)
-
-        # Test vectorization of log
-        base_point = gs.stack([base_point] * n_samples, axis=0)
-        point = gs.stack([point] * n_samples, axis=0)
-        expected = gs.stack([one_log] * n_samples, axis=0)
-
-        log = h5_metric.log(point=point, base_point=base_point)
-        result = log
-
-        self.assertAllClose(gs.shape(result), (n_samples, dim + 1))
-        self.assertAllClose(result, expected)
-
-        result = h5_metric.exp(tangent_vec=log, base_point=base_point)
-        expected = point
+            return self.generate_tests(smoke_data)
+
+        def extrinsic_to_intrinsic_coords_rasises_data(self):
+            smoke_data = [
+                dict(
+                    dim=3,
+                    point=gs.array([-1.0, 1.0, 0.0, 0.0]),
+                    expected=pytest.raises(ValueError),
+                )
+            ]
+            return self.generate_tests(smoke_data)
+
+        def random_point_belongs_data(self):
+            smoke_space_args_list = [(2,), (3,)]
+            smoke_n_points_list = [1, 2]
+            belongs_atol = gs.atol * 100000
+            return self._random_point_belongs_data(
+                smoke_space_args_list,
+                smoke_n_points_list,
+                self.space_args_list,
+                self.n_points_list,
+                belongs_atol,
+            )
+
+        def to_tangent_is_tangent_data(self):
+
+            is_tangent_atol = gs.atol * 1000
+
+            return self._to_tangent_is_tangent_data(
+                Hyperboloid,
+                self.space_args_list,
+                self.shape_list,
+                self.n_vecs_list,
+                is_tangent_atol,
+            )
+
+        def projection_belongs_data(self):
+            return self._projection_belongs_data(
+                self.space_args_list,
+                self.shape_list,
+                self.n_samples_list,
+                belongs_atol=gs.atol * 1000,
```

Comment:
Why 1000? it was 100 in the former tests

Reply:
it was being tested for fixed dimension. now more dimensions